src/static/helpers/slasHelper.ts

```diff
@@ -147,6 +147,70 @@ export async function authorize(
   return {url: redirectUrlString, ...getCodeAndUsidFromUrl(redirectUrlString)};
 }
 
+/**
+ * Wrapper for the /token endpoint. Depending on the parameters provided, will set up different grant types.
+ * @param slasClient a configured instance of the ShopperLogin SDK client
+ */
+export async function getAccessToken(
+  slasClient: ShopperLogin<{
+  shortCode: string;
+  organizationId: string;
+  clientId: string;
+  siteId: string;
+}>,
+parameters: {
+  redirectURI: string;
+  refreshToken?: string;
+  codeVerifier?: string;
+  code?: string;
+  usid?: string;
+},
+credentials?: {
+  client_id: string;
+  client_secret: string;
+}
```

Comment:
In the PWA Kit, we're using a proxy to intercept the call to SLAS and inject the client secret there. So PWA Kit does not set this arg.

However, this arg is here to enable non-PWA Kit projects that may want to use a private client.